__tests__/utils/errors.test.ts

```diff
@@ -2,7 +2,8 @@ import { RPC, RpcError } from '../../src';
 
 describe('Error utility tests', () => {
   test('RpcError', () => {
-    const baseError: RPC.Errors.UNEXPECTED_ERROR = {
+    // TODO: @penovicp types 0.8 exporta u top level sve 0.7 exporta po komponentama, neznan jeli treba minjat test i mod da ocukuje oba tipa
```

Comment:
The type export change shouldn't be an issue. The RPC error utility relies on the mappings set up in `src/types/errors.ts`, so when the mapping is updated the utility should work regardless of how the underlying types are imported. It should work even without an update since the 0.7 errors are a subset of the 0.8 errors, the difference is that without it the new 0.8 errors wouldn't be recognised for type narrowing.